spot-client/src/spot-tv/ui/components/meeting-frame/meeting-frame.js

```diff
@@ -541,12 +566,19 @@ export class MeetingFrame extends React.Component {
      *
      * @param {Object} event - An object with participant information.
      * @param {string} event.id - The jitsi-meet ID of the participant.
-     * @param {string} even.displayName - The current name of the participant.
+     * @param {string} event.displayName - The current name of the participant.
      * @private
      * @returns {void}
      */
     _onParticipantJoined({ id, displayName }) {
         this._participants.set(id, displayName);
+        const newMax = this._participants.size + 1;
+
+        if (this.state.maxParticipantCount < newMax) {
+            this.setState({
+                maxParticipantCount: this._participants.size + 1
+            });
+        }
```

Comment:
I don't think you need to set this on state. state is usually used for triggering a re-render, whereas these don't need to.

Reply:
🤦‍♂